stuff/cron/update_ranks.py

```diff
@@ -662,6 +663,25 @@ def update_coder_of_the_month_candidates(
                         row['score'],
                         row['ProblemsSolved']
                     ))
+        cur.execute('''
+                    INSERT INTO
+                        `Notifications` (
+                            `user_id`,
+                            `contents`
+                        )
+                    VALUES (
+                        %s,
+                        %s
+                    );
+                    ''',
+                    (
+                        row['user_id'],
+                        json.dumps({'type': 'coder-of-the-month',
+                                    'body': {'localizationString':
+                                             'coderOfTheMonthNotice',
+                                             'localizationParams':
+                                             {'username': row['username']},
+                                             'iconUrl': '/media/info.png'}})))
```

Comment:
nit:

```suggestion
        cur.execute(
            '''
            INSERT INTO
                `Notifications` (
                    `user_id`,
                    `contents`
                )
            VALUES (
                %s,
                %s
            );
            ''',
            (
                row['user_id'],
                json.dumps({
                    'type': 'coder-of-the-month',
                    'body': {
                        'localizationString': 'coderOfTheMonthNotice',
                        'localizationParams': {
                            'username': row['username'],
                        },
                        'iconUrl': '/media/info.png',
                    },
                }),
            ),
        )
```

así está más regular el espaciado y se entiende mejor la estructura del JSON.